src/layout/workspace.rs

```diff
@@ -1536,6 +1536,28 @@ impl<W: LayoutElement> Workspace<W> {
         self.activate_column(self.columns.len() - 1);
     }
 
+    pub fn focus_column_right_or_first(&mut self) {
+        if self.columns.is_empty() {
+            return;
+        }
+
+        let column_idx = (self.active_column_idx + 1) % self.columns.len();
+        self.activate_column(column_idx);
+    }
+
+    pub fn focus_column_left_or_last(&mut self) {
+        if self.columns.is_empty() {
+            return;
+        }
+
+        let column_idx = if self.active_column_idx == 0 {
+            self.columns.len() - 1
+        } else {
+            self.active_column_idx - 1
+        };
```

Comment:
Could also do
```rust
let column_idx = match self.active_column_idx.overflowing_sub(1) {
    (i, false) => i,
    (_, true) => self.columns.len() - 1,
}
```
I prefer the `if` though

Reply:
Yeah, if is good.